spanner/changestreams/src/test/java/com/example/spanner/changestreams/ChangeStreamSampleIT.java

```diff
@@ -73,7 +74,8 @@ public void setUp() {
     dbId = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
     dbClient.dropDatabase(dbId.getInstanceId().getInstance(), dbId.getDatabase());
     try {
-      dbClient.createDatabase(instanceId, databaseId, Collections.emptyList()).get();
+      dbClient.createDatabase(instanceId, databaseId, Collections.emptyList())
+          .get(15, TimeUnit.MINUTES);
```

Comment:
I think our default for tests is 10, so you might wish to lower this as well or increase kokoro's timeout.